core-bundle/contao/drivers/DC_Table.php

```diff
@@ -6792,7 +6792,7 @@ protected function getClipboardPermission(string $mode, int $id, array|null $new
 			'cut',
 			'cutAll' => new UpdateAction($this->strTable, $this->getCurrentRecord($id, $this->strTable), array_replace(array('sorting' => null), (array) $new)),
 			'copy',
-			'copyAll' => new CreateAction($this->strTable, array_replace($this->getCurrentRecord($id, $this->strTable), array('tstamp' => null, 'sorting' => null), (array) $new))
+			'copyAll' => new CreateAction($this->strTable, array_replace($this->getCurrentRecord($id, $this->strTable) ?? [], array('tstamp' => null, 'sorting' => null), (array) $new))
```

Comment:
```suggestion
			'copyAll' => new CreateAction($this->strTable, $this->getCurrentRecord($id, $this->strTable), array_replace(array('tstamp' => null, 'sorting' => null), (array) $new))
```

Isn‘t this the correct fix? At least this is how the code works in line 6793. @aschempp /cc

Reply:
@fritzmg @aschempp /ping

Reply:
I don't think so, but @aschempp will have to confirm.

Reply:
@leofeyer `UpdateAction` requires the table, the current record and the new record, so we pass

```php
new UpdateAction(
    $this->strTable,
    $this->getCurrentRecord($id, $this->strTable),
    array_replace(array('sorting' => null), (array) $new),
);
```

The `CreateAction` requires the table and the new record, so we pass:

```php
new CreateAction(
    $this->strTable,
    array_replace($this->getCurrentRecord($id, $this->strTable), array('tstamp' => null, 'sorting' => null), (array) $new))
);
```

Passing

```php
new CreateAction(
    $this->strTable, 
    $this->getCurrentRecord($id, $this->strTable), 
    array_replace(array('tstamp' => null, 'sorting' => null), (array) $new)
);
```

as per your suggestion would lead to an error, as `CreateAction` only takes two constructor arguments.

Reply:
Yes, you are right. We should probably cast to an array here then: `(array) $this->getCurrentRecord($id, $this->strTable)`